loops.js

```diff
@@ -50,4 +69,14 @@
 // >> 8
 // >> 9
 // ==========================================
+console.log("Bonusopdracht:");
 
+for (let i = 0; i < 10; i++) {
+    if (i >= 6) {
+        console.log(">> " + i);
+    } else if (i >= 3 ) {
+        console.log("> " + i);
+    } else {
+        console.log(i);
+    }
+}
```

Comment:
Anders geschreven dan het antwoordmodel, maar logt de uitkomst zoals verwacht, verder ziet het er keurig uit. [de "}" opmerking is op dit moment boventallig].